wpimath/src/main/native/include/frc/controller/SimpleMotorFeedforward.h

```diff
@@ -201,20 +201,41 @@ class SimpleMotorFeedforward {
    */
   constexpr units::volt_t GetKs() const { return kS; }
 
+  /**
+   * Returns the static gain.
+   *
+   * @param kS The static gain.
+   */
+  constexpr void SetKs(units::volt_t kS) { this->kS = kS; }
+
   /**
    * Returns the velocity gain.
    *
    * @return The velocity gain.
    */
   constexpr units::unit_t<kv_unit> GetKv() const { return kV; }
 
+  /**
+   * Returns the velocity gain.
+   *
+   * @param kV The velocity gain.
+   */
+  constexpr void SetKv(units::unit_t<kv_unit> kV) const { this->kV = kV; }
+
   /**
    * Returns the acceleration gain.
    *
    * @return The acceleration gain.
    */
   constexpr units::unit_t<ka_unit> GetKa() const { return kA; }
 
+  /**
+   * Returns the acceleration gain.
+   *
+   * @param kA The acceleration gain.
+   */
+  constexpr void SetKa(units::unit_t<ka_unit> kA) const { this->kA = kA; }
```

Comment:
const qualification is only for member functions which don't mutate the object. Since this is a setter, remove the const.